src/pages/Details.js

```diff
@@ -2,83 +2,67 @@ import React from 'react';
 import '../styling/Details.css'
 import Source from '../components/Source'
 import { dateToString } from '../util';
-import Carousel from 'react-bootstrap/Carousel' 
+import Carousel from 'react-bootstrap/Carousel'
 import logo from '../images/cwb-logo-reverse-w-tagline 1.png';
 import axios from 'axios';
 import { withRouter } from 'react-router-dom';
-
-
+ 
+ 
 class Details extends React.Component {
-   componentDidMount() {
-    const { match: { params } } = this.props;
-  
+  componentDidMount() {
+   const { match: { params } } = this.props;
     axios.get(`/details/${params.userId}`)
-      .then(({ data: user }) => {
-        console.log('user', user);
-  
+     .then(({ data: user }) => {
         this.setState({ user });
-
-      });
-  }
-  //console.log({this.props.location.state})
-  //let currentSubmission;
-  //var currentSubmission = props.location.state.mySubmission;
-  //let var = this.props.location.state;
+ 
+     });
+ }
+
+  render() {
+   console.log("hello");
```

Comment:
remove console.log statements before merging to master (might help to grep for 'console.log' in the src directory before opening PR to make sure there aren't any)